Unix/samples/Providers/Indication/Alert/XYZ_DiskFault.c

```diff
@@ -75,7 +75,7 @@ void MI_CALL XYZ_DiskFault_EnableIndications(
     const MI_Char* className)
 {
     /* TODO: store indicationsContext for posting indication usage */
-    /* NOTE:Call one of following functions if and ONLY if encount termination error,
+    /* NOTE:Call one of following functions if and ONLY if encounter termination error,
```

Comment:
---
Beyond here, the version of the diff I'm looking at isn't automatically expanded.